esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/opt/rose-suite-metoffice.conf

```
@@ -2,6 +2,8 @@
 DRS_CMIP6="BADC"
 DRS_OBS4MIPS="default"
 MODULE_NAME="scitools/community/esmvaltool/2.6.0"
-ROOTPATH_CMIP6="/project/champ/data/CMIP6"
+ROOTPATH_CMIP6="/project/cma/esmvaltool/CMIP6"
```

This change must be reverted (the `/project/champ/` path is the one we want to use):
```suggestion
ROOTPATH_CMIP6="/project/champ/data/CMIP6"
```

---

This was to test to see if this would work, as the original one didn't. More investigation needed.

---

This was also addressed in 9cab64c